src/Playwright.TestingHarnessTest/SimpleSmokeTests.NUnit.cs

```diff
@@ -0,0 +1,22 @@
+using System.Threading.Tasks;
+using Microsoft.Playwright.NUnit;
+using NUnit.Framework;
+
+namespace Playwright.Tests.MSTest
+{
+
+    public class SimpleSmokeTestsNUnit : PageTest
+    {
+        [Test]
+        public async Task ShouldOpenPlaywright()
+        {
+            Assert.IsNotNull(Page);
+            await Page.GotoAsync("https://www.playwright.dev");
```

Comment:
Could this be instead either be setContent or evaluate 1+1==2? Unit tests should work without internet.

Reply:
So, a couple arguments against that:

1) This isn't really a critical part of our test suite.
2) I want these particular "smoke" tests to accurately represent a common customer scenario, which I feel is usually opening a site and checking for some parameters.  I want this suite to grow a little bit throughout time, but again, not testing *our* functionality as much as giving a few real-world examples of where Playwright is useful.
3) In this particular example, considering where they'll be run, I don't mind that much :)